src/main/markdown/eco/add-lib.md

```diff
@@ -0,0 +1,26 @@
+# Update Information
+
+We will add or update information on request. 
+
+## Request for adding Libraries to the Eco System site
+
+The GWT eco system is a living system. New libs will be created, other will be end of life. Also we know, that there are
+much more libraries outside. If you think, one library not listed here is worth to be mentioned, [open an issue
+on our tracker](https://github.com/gwtproject/gwt-site/issues/new/choose). This will trigger a process to get mentioned in the eco system content.
+
+We will expect:
+
+* The Issue must be created by a contributor of the library
```

Comment:
What are the concerns here, that someone else might publicize a library that wasn't ready to be showcased?

Reply:
Ok, remove it.

Done